src/main/java/org/carlmontrobotics/RobotContainer.java

```diff
@@ -57,10 +66,31 @@
 import edu.wpi.first.wpilibj2.command.PrintCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 
+// control bindings
+
 // control bindings
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import edu.wpi.first.wpilibj2.command.button.POVButton;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+//pathplanner
+import com.pathplanner.lib.auto.NamedCommands;
+import com.pathplanner.lib.commands.PathPlannerAuto;
+import com.pathplanner.lib.path.GoalEndState;
+import com.pathplanner.lib.path.PathConstraints;
+import com.pathplanner.lib.path.PathPlannerPath;
+import static com.pathplanner.lib.auto.AutoBuilder.*;
+import com.pathplanner.lib.auto.AutoBuilder;
+import org.carlmontrobotics.subsystems.Led;
+
+//java
+import java.util.function.DoubleSupplier;
+import java.util.function.BooleanSupplier;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
```

Comment:
Presumably unneeded since the only real change to this file seems to be the addition of the call to `SensorFactory.configureCamera()`.